backend/src/main/java/de/grimsi/gameyfin/service/DownloadService.java

```diff
@@ -87,6 +109,7 @@ private void sendGamefilesAsZipToClient(Path path, OutputStream outputStream) {
                 @SneakyThrows
                 public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
                     zos.putNextEntry(new ZipEntry(path.relativize(file).toString()));
+                    log.info("Adding file {} to archive...", file);
```

Comment:
This should be a DEBUG level log because it's not really important to log every file in the archive (errors will load to an exception and get logged anyway).

Reply:
Yeah makes sense! Changed that :+1: